### DISCUSSION TITLE
query

### DISCUSSION THREAD

**Discussion by noorunisaa** [AammarTufail/python_ka_chilla2023#121] [Q&A]
Can anyone please help me why my script.py isn't showing in cmd? I am a beginner and just done with the day 1 class of pkc2023.
![Screenshot (62)](https://github.com/AammarTufail/python_ka_chilla2023/assets/155584170/76cab33b-1a7e-4698-b943-b9d8b635d419)

**Comment by AmnaSaeed47** [comment]
It means this file is not exist in the folder



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
01/04/24,
04:05:41 PM
